app/assets/stylesheets/components/page_specific/home_beta/_seasonal_spotlight.scss

```diff
@@ -0,0 +1,29 @@
+.seasonal-spotlight {
+  @include column(12);
+  margin-top: $baseline-unit*2;
+  margin-bottom: $baseline-unit*2;
+  background-size: cover;
+  position: relative;
+  height: 300px;
+
+  @include respond-to($mq-m) {
+    @include column(6);
+    height: 339px;
```

Comment:
Could this usefully use flexbox to keep the heights of the two components aligned?